kong/dao/schemas/apis.lua

```diff
@@ -16,11 +17,38 @@ local function validate_target_url(value)
   return false, "Invalid target URL"
 end
 
+local function check_public_dns_and_path(value, api_t)
+  local public_dns = type(api_t.public_dns) == "string" and stringy.strip(api_t.public_dns) or ""
+  local path = type(api_t.path) == "string" and stringy.strip(api_t.path) or ""
```

Comment:
Yeah sounds good, I can add this to the list of improvements. Check for alphanumeric characters only (since we don't support regex yet)